packages/react/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,134 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import {
+  UIComponent,
+  UIComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  customPropTypes,
+  childrenExist,
+  rtlTextContainer,
+} from '../../lib'
+import { RenderResultConfig } from 'src/lib/renderComponent'
+import { defaultBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+import { ComponentEventHandler, ReactProps, ShorthandValue } from '../../types'
+import { Box, Icon } from '../..'
+
+export type AlertSlotClassNames = Record<'content' | 'closeIcon', string>
+
+export interface AlertProps extends UIComponentProps, ContentComponentProps {
```

Comment:
I know I am jumping a bit late on this, but would expect the Alert component to have title as well.

Reply:
We decided to skip this for first implementation. There is also open discussion for `title` vs `header`...

Reply:
we thought `content` is enough for v1, we plan to add it in v2
see #1061 description and

> skip header and statusIcon/icon for initial implementation
from https://github.com/stardust-ui/react/issues/1061#issuecomment-473248610

@mnajdova we can reopen the discussion now if you feel there is a need to support this even in v1; fyi @layershifter

Reply:
let's not forget `title` is an HTML attribute

Reply:
Let's leave it for a next iteration (I agree header is better name)